app/views/staff/proposals/show.html.haml

```diff
@@ -43,7 +43,7 @@
       = render partial: 'proposals/confirmation_notes', locals: {proposal: proposal, notes: proposal.confirmation_notes}
 
     .col-md-4
-      %h3 Tags
+      %h3 Review Tags
```

Comment:
It might make sense wrap both the heading and the partial in a conditional. If the organizer has decided not to use tags, there don't need to be any indications that there *could* be tags show here.